feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/PreviewScreen.kt

```diff
@@ -109,27 +114,46 @@ fun PreviewScreen(
     if (previewUiState.cameraState == CameraState.NOT_READY) {
         Text(text = stringResource(R.string.camera_not_ready))
     } else if (previewUiState.cameraState == CameraState.READY) {
-        Box(
-            modifier = Modifier
-                .fillMaxSize()
-                .transformable(state = transformableState)
-                .pointerInput(Unit) {
-                    detectTapGestures(
-                        onDoubleTap = { offset ->
-                            Log.d(TAG, "onDoubleTap $offset")
-                            viewModel.flipCamera()
-                        }
-                    )
-                }
+        BoxWithConstraints(
+            Modifier.background(Color.Black),
+            contentAlignment = Alignment.Center
         ) {
-            CameraPreview(
+            var aspectRatioEnum by remember {
+                mutableStateOf(previewUiState.currentCameraSettings.aspect_ratio)
+            }
+            val maxAspectRatio: Float = maxWidth / maxHeight
+            val aspectRatio: Float = aspectRatioEnum.ratio.toFloat()
```

Comment:
referencing above comment, can just use `previewUiState.currentCameraSettings.aspect_ratio` here instead of the aspectRatioEnum